src/Pdf.php

```diff
@@ -70,9 +72,16 @@ public function setTimeout($timeout) {
         return $this;
     }
 
+    public function setEnv(array $env)
+    {
+        $this->env = $env;
+        return $this;
+    }
+
     public function text(): string
```

Comment:
Instead of introducing a seperate method for this, I'd rather have a callable optionally being passed to `text`. If a callable is given, it should receive the `$process`. This way people can customize the process however they want.

Could you adjust the PR to support this behaviour?